envYaml/common/common.yaml

```diff
@@ -35,4 +31,29 @@ endPoints:
   hrmsSearch: egov-hrms/employees/_search
   hrmsUpdate: egov-hrms/employees/_update
  accessControl:
-  search: access/v1/actions/mdms/_get
\ No newline at end of file
+  search: access/v1/actions/mdms/_get
+  invalidSearchMethod: access/v1/actions/mdms/_create
+ localizationSearch: localization/messages/v1/_search
+ localizationV2Search: localization/messages/v2/_search
+ localizationUpdate: localization/messages/v1/_update
+ localizationDelete: localization/messages/v1/_delete
+ localizationCreate: localization/messages/v1/_create
+ localizationUpsert: localization/messages/v1/_upsert
+ authToken: user/oauth/token
+ userOtpSend: user-otp/v1/_send
+ fileStoreCreate: filestore/v1/files
+ fileStoreGet: filestore/v1/files/url
+ createUser: user/users/_createnovalidate
+ searchUser: user/_search
+ searchLocation: egov-location/location/v11/boundarys/_search
+ idGeneate: egov-idgen/id/_generate
+ searchMdms: egov-mdms-service/v1/_search
+ invalidSearchMdms: egov-mdms-service/v2/_search
+ searcher: egov-searcher/bill-genie/billswithaddranduser/_get
+ searcherWS: egov-searcher/bill-genie/waterbills/_get
+ searcherSW: egov-searcher/bill-genie/seweragebills/_get
+ metadataGetReport: report/pgr/DescriptionReport/metadata/_get
+ getReport: report/pgr/DescriptionReport/_get
+ updateUser: user/profile/_update
+ updatePassword: user/password/_update
+ updatePasswordNoLogin: user/password/nologin/_update
```

Comment:
please remove these

Reply:
apart from what u have added

Reply:
@yatheendra-egov I have addressed this comment and updated the PR as well.